tests/controllers/months.test.js

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable max-len */
+const chai = require('chai')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
+const models = require('../../models')
+const {
+  after, afterEach, beforeEach, describe, it,
+} = require('mocha')
+const { singleMonth, monthList } = require('../mocks/months')
+const {
+  getAllMonths,
+  getMonthByName,
+  getMonthBySlug,
+} = require('../../controllers/months')
+
+chai.use(sinonChai)
+const { expect } = chai
+
+describe('Controllers - months', () => {
+  let response
+  let sandbox
+  let stubbedMonthsFindOne
+  let stubbedMonthsFindAll
+  let stubbedStatusSend
+
+  beforeEach(() => {
+    sandbox = sinon.createSandbox()
+
+    stubbedStatusSend = sandbox.stub()
+
+    response = {
+      send: sandbox.stub(),
+      status: sandbox.stub().returns({ send: stubbedStatusSend }),
+      sendStatus: sandbox.stub(),
+    }
+
+    stubbedMonthsFindOne = sandbox.stub(models.Months, 'findOne')
+    stubbedMonthsFindAll = sandbox.stub(models.Months, 'findAll')
+  })
+
+  after(() => {
+    sandbox.reset()
+  })
+
+  afterEach(() => {
+    sandbox.restore
+  })
+
+  describe('Months', () => {
+    describe('getAllMonths', () => {
+      it('retrieves a list of months', async () => {
+        const stubbedMonthsFindAll = sinon.stub(models.Months, 'findAll').returns(monthList)
```

Comment:
if you create a stub inside a test it will not be available in any other tests. Go back and review that material on stubbing, before, etc.